### PR TITLE
README.md: Update README

### PR DESCRIPTION
**Pull request by GordanM**
Reformat the README in markdown format, fix broken links and update
repository information and description.

Signed-off-by: Gordan Markuš <gordan.markus@pelagicore.com>

### CODE REVIEW

**Review by frznlogic — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+Master location for GENIVI Persistence Common Object: https://github.com/GENIVI/persistence-common-object
+
+## License
+
+For licensing information see the COPYING file, distributed along with this project.
+
+## Build Dependencies and Instructions
+
+The Persistence Common Object has dependencies on the itzam library. The Persistence Common Object specifies an API to access the key/value and configuration data of GENIVI's persistent data handling.
```
Comment (gunnarx) [inline README.md]: Itzam is no longer the preferred database, right?
So it should no longer have dependencies on it, I guess?

**Review by gunnarx — COMMENTED** [review]
Oh I didn't mean to start a review - too formal.  Just asking...

File: README.md (hunk 1 above)
Comment (GordanM) [reply README.md]: You are correct @gunnarx, I will remove that line and write down other non-trivial dependencies if I find any.

Comment (GordanM) [reply README.md]: According to Makefile.am the non-trivial deps I could fine where the: dlt-daemon, dbus libs and gobject libs, though the last to ones could be considered trivial as well. 
 
libpers_common_la_LIBADD = \
                            $(DLT_LIBS)             \
                            $(GIO_LIBS)             \
                            $(GIO_UNIX_LIBS)        \
                            $(GLIB_LIBS)            \
                            $(DBUS_LIBS)            \
                            $(GOBJECT_LIBS)

if HAVE_ITZAM
libpers_common_la_LIBADD += \
                             $(ITZAM_LIBS)
endif

if HAVE_SQLITE
libpers_common_la_LIBADD += \
                             $(SQLITE_LIBS)
endif

if HAVE_RAWDB
libpers_common_la_LIBADD += \
                             $(RAWDB_LIBS)
endif

**Pull request merged**